doc/client-setup.md

```diff
@@ -0,0 +1,120 @@
+# Client Application Setup
+
+To seamlessly integrate a client application with a host, you will need to alter the client
+application's default behavior in a few ways. Detailed instructions are below. _Navigation is
+the trickiest item to get correct. Be sure to read that section carefully._
+
+## Application Bootstrap
+
+Early in the application bootstrap process, you will need to create a [client instance](../classes/client.client-1.html), add an event
+listener for environment data passed from the host application, and start the client.
+
+[Environment data](../interfaces/client.envdata.html) is data set in the host application that is static over time, but that clients will
+need to function, such as the user-selected locale. It supports both pre-defined common/required
+options and custom data.
+
+Calling [`start`](../classes/client.client-1.html#start) on the client sets up messaging
+listeners, and sends a signal to the host application to inform it that your application is
+ready to start recieving messages.
+
+The ifc phase of a typical client bootstrap might look like this:
```

Comment:
When I first read this I was trying to figure out what `ifc phase` was, might be worth saying `iframe-coordinator (ifc) phase` since it is the first time referring to the `ifc` abbreviation in this doc. Maybe `ifc phase` is obvious enough to other readers, though.